entrypoint.sh

```diff
@@ -43,16 +43,21 @@ elif [[ "$INPUT_WRANGLERVERSION" == 1* ]]; then
   npm i -g "@cloudflare/wrangler@$INPUT_WRANGLERVERSION"
   WRANGLER_VERSION=1
 
-# Else install Wrangler 2
+# If Wrangler version starts with 2 then install wrangler v2
+elif [[ "$INPUT_WRANGLERVERSION" == 2* ]]; then
+  npm i -g "@cloudflare/wrangler@$INPUT_WRANGLERVERSION"
```

Comment:
This block was only done for Wrangler 1 because the package changed. We do not need it (it also would not work since that's the wrangler 1 package, wrangler 2 is just `wrangler`)